tensordict/tensorclass.py

```diff
@@ -172,9 +173,12 @@ def __torch_function__(
         args: tuple[Any, ...] = (),
         kwargs: dict[str, Any] | None = None,
     ) -> Callable:
-        if func not in _TD_PASS_THROUGH or not all(
```

Comment:
Doesn't this work? I'd expect it to work.

Reply:
Yep I think i had an error with that at some point but I can't reproduce it now, reverting